docs/Execution.md

```diff
@@ -33,7 +33,11 @@ QUIC_STATUS
     );
 ```
 
-The application must register a callback handler for every MsQuic object it creates. This handler must manage all the events MsQuic may indicate for that object. The handler must also return a status for each event indicating to MsQuic how the event was handled. This returned status is often success/failure, but sometimes indicates MsQuic that further processing is required.
+The application must register a callback handler for every MsQuic object it creates, as well as for some objects automatically created by MsQuic (TODO: add reference).
```

Comment:
For reference, you can mention peer initiated streams, or connections (for server side).